src/graphnet/models/standard_model.py

```diff
@@ -50,11 +51,24 @@ def __init__(
         assert isinstance(tasks, (list, tuple))
         assert all(isinstance(task, StandardLearnedTask) for task in tasks)
         assert isinstance(graph_definition, GraphDefinition)
-        assert isinstance(gnn, GNN)
+
+        # deprecation warnings
+        if (architecture is None) & (gnn is not None):
+            architecture = gnn
+            # Code continues after warning
+            self.warning(
```

Comment:
Any reason why you're using the logger and not ```warnings.warn(""Argument `gnn` will be deprecated in GraphNeT 2.0. Please use `architecture` instead., DeprecationWarning)```. To me this feels like an avoidable warning made by a user. See [python documentation on logging](https://docs.python.org/3/howto/logging.html)